operators/synapse-helm/0.0.1/scorecard/config.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: scorecard.operatorframework.io/v1alpha3
```

Comment:
@mgoerens please move file `operators/synapse-helm/0.0.1/scorecard/config.yaml` to `operators/synapse-helm/0.0.1/tests/scorecard/config.yaml`

Reply:
Thanks Martin, done!